### PR TITLE
Extended footer is added

### PR DESCRIPTION
**Pull request by AbhijitVyas**

### CODE REVIEW

**Comment by AbhijitVyas** [comment]
Hi @daniel86 ,
Please review this PR. For some strange reasons I can not select you as reviewer in Reviewers section.
Best,
Abhijit

File: templates/base.html [hunk 1]
```diff
@@ -122,12 +125,202 @@
...
+        <!-- Social buttons -->
+    </div>
+
+
 </footer>
 <script type="text/javascript" src="{{ url_for('static', filename='js/form-js/copyright_year.js') }}"></script>
+<script>
+;(function () {
+  $(document).ready(function () {
+    var $button, $extended_footer
```
Comment (daniel86) [inline templates/base.html]: missing semicolon. Also in lines below

Comment (daniel86) [inline templates/base.html]: strange variable name starting with "$", could you remove it?

File: templates/base.html [hunk 2]
```diff
@@ -122,12 +125,202 @@
...
+    $button = $('#toggle_button')
```
Comment (daniel86) [inline templates/base.html]: I would write the *var* statement here, like that:
```
var button = $(...);
```

File: templates/base.html [hunk 3]
```diff
@@ -122,12 +125,202 @@
...
+      if ($extended_footer.attr('hidden') === 'hidden') {
+        $extended_footer.removeAttr('hidden')
+        return $button.attr('aria-expanded', true)
+      } else {
+        $extended_footer.attr('hidden', true)
+        return $button.attr('aria-expanded', false)
+      }
+    })
+  })
+}.call(this))
```
Comment (daniel86) [inline templates/base.html]: here is also semicolon missing

File: templates/flask_user/register.html [hunk 4]
```diff
@@ -90,6 +89,8 @@
                         {% endif %}
                     </div>
                 </div>
+                </div>
```
Comment (daniel86) [inline templates/flask_user/register.html]: the indentation is wrong here, could you fix it?

**Review by daniel86 — CHANGES_REQUESTED** [review]

**Pull request merged**